Library/Homebrew/formula_installer.rb

```diff
@@ -344,7 +344,15 @@ def expand_dependencies(deps)
     inherited_options = {}
 
     expanded_deps = Dependency.expand(formula, deps) do |dependent, dep|
-      options = inherited_options[dep.name] = inherited_options_for(dep)
+      additional_inherited = inherited_options_for(dep)
+      unless additional_inherited.nil?
```

Comment:
`if additional_inherited` is simpler. I also personally hate `unless` .. `else` as I find the double-negatives hard to parse.